public/partials/demand_new.html

```diff
@@ -20,13 +21,19 @@
                 No matches found for "{{searchText}}".
             </md-not-found>
         </md-autocomplete>
+
+        <md-input-container ng-if="edit_demand">
+            <label>Crop</label>
+            <input readonly ng-model="demand.cr_crop.cr_crop_name" type="text"/>
```

Comment:
@nick22891 why is the `readonly` attribute here?  I thought the purpose is to edit the demand.